cppguide.html

```diff
@@ -1466,9 +1466,9 @@ <h3 id="Structs_vs._Classes">Structs vs. Classes</h3>
 be present; however, these methods must not require or enforce any
 invariants.</p>
 
-<p>If more functionality or invariants are required, a
-<code>class</code> is more appropriate. If in doubt, make
-it a <code>class</code>.</p>
+<p>If more functionality or invariants are required, or struct has wide visibility and expected to
```

Comment:
This grammar seems problematic

Reply:
I agree, but that should be fixed with an internal update and then syncing the public style guide to it. If you get around to it before me, let me know and I can update the PR.

Reply:
I'm not going to get to this this week.